utility/cli_message_examples_reprex.md

```diff
@@ -157,11 +212,72 @@ missing_list_col_supertbl <- tibble(redcap_data = list(), redcap_metadata = 123)
   as_supertbl()
 make_labelled(missing_list_col_supertbl)
 #> Error in `make_labelled()`:
-#> ✖ You've supplied `<redcap_supertbl[,2]>` for `supertbl` which is not a
-#>   valid value
+#> ✖ You've supplied `<rdcp_spr[,2]>` for `supertbl` which is not a valid
+#>   value
 #> ! `supertbl$redcap_metadata` must be of type 'list'
 #> ℹ `supertbl` must be a REDCapTidieR supertibble, generated using
 #>   `read_redcap()`
+
+# add_skimr_metadata
+
+mtcars %>% add_skimr_metadata()
+#> Error in `add_skimr_metadata()`:
+#> ✖ You've supplied `<df[,11]>` for `supertbl` which is not a valid value
+#> ! Must be of class <redcap_supertbl>
+#> ℹ `supertbl` must be a REDCapTidieR supertibble, generated using
+#>   `read_redcap()`
+
+# write_redcap_xlsx
+
+withr::with_tempdir({
+  dir <- getwd()
+  filepath <- paste0(dir, "/temp.csv")
+  REDCapTidieR:::check_file_exists(file = filepath, overwrite = FALSE)
+})
+
+withr::with_tempdir({
+  dir <- getwd()
+  tempfile <- write.csv(x = mtcars, file = "temp.csv")
+  filepath <- paste0(dir, "/temp.csv")
+  REDCapTidieR:::check_file_exists(file = filepath, overwrite = FALSE)
+})
+#> Error:
+#> ✖ File
+#>   ''/private/var/folders/qc/mmjjyjq50530z9r_7mfqcqfhxkkk67/T/Rtmp9DL6jB/file35a532406eec/temp.csv''
+#>   already exists.
+#> ℹ Set `overwrite = TRUE` to overwrite your file.
```

Comment:
(i) Overwriting files is disabled by default. Set `overwrite = TRUE` to overwrite existing file.